actors/builtin/miner/miner_state.go

```diff
@@ -50,18 +50,12 @@ type MinerInfo struct {
 }
 
 type PoStState struct {
-	// Epoch of the last succesful PoSt, either election post or surprise post.
-	LastSuccessfulPoSt abi.ChainEpoch
-
-	// If >= 0 miner has been challenged and not yet responded successfully.
-	// SurprisePoSt challenge state: The miner has not submitted timely ElectionPoSts,
-	// and as a result, the system has fallen back to proving storage via SurprisePoSt.
-	//  `epochUndefined` if not currently challeneged.
-	SurpriseChallengeEpoch abi.ChainEpoch
+	// Epoch that starts the current proving period
```

Comment:
Maybe note also that this is the start of the challenge window (if I'm interpreting correctly that the window is at the start of the period).